vulyk/models/task_types.py

```diff
@@ -290,6 +300,13 @@ def _del_work_session(self, task, user):
         if rs.count() > 0:
             rs.first().delete()
         else:
-            msg = "No session was found for {0} & {1}".format(user, task.id)
+            msg = 'No session was found for {0} & {1}'.format(user, task.id)
 
             raise WorkSessionLookUpError(msg)
+
+    def to_dict(self):
+        return {
```

Comment:
It was created as a overridable representation method. In case if smb would meet a need to change the view somehow.
And if we create this method implicitly - why not to cut off some peels?